### PR TITLE
add gazebo simulation

### PR DESCRIPTION
**Pull request by pkowalsk1**

### CODE REVIEW

**Review by KmakD — CHANGES_REQUESTED** [review]

File: demo/compose.ros2-bridge.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+services:
+  
+  bridge:
+    image: husarion/ros:galactic-ros1-bridge
+    restart: always
+    network_mode: host
+    ipc: host
```
[the remark this answers is not in the document]
Comment (pkowalsk1) [reply demo/compose.ros2-bridge.yaml:7]: This is a widely used practice with ros2 containers. Check for example this [link](https://github.com/husarion/rosbot-docker/blob/c64a23e9ec78f738105cdffcc0a8f897d43a5268/demo/minimal_setup/compose.minimal.ros2.yaml#L10) and [this](https://github.com/husarion/rosbot-docker/blob/ros1/demo/autonomous_navigation_mapping/compose.rosbot.lan.yaml).

File: Dockerfile.simulation [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+RUN apt-get update  && \
+    apt-get install -y \
+        git \
+        python3-dev \
+        python3-pip && \
+    pip3 install \
+        rosdep && \
+    git clone -b ros1 https://github.com/husarion/ros_components_description.git src/ros_components_description && \
+    git clone -b mv-panther-gazebo https://github.com/husarion/panther_ros.git && \
```
Comment (KmakD) [inline Dockerfile.simulation:15]:
```suggestion
    git clone https://github.com/husarion/panther_ros.git && \
```

File: demo/simulation/compose.simulation-gpu.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+services:
+
+  panther_gazebo:
+    # image: husarion/panther:noetic-simulation
+    build:
+      context: ./../..
+      dockerfile: Dockerfile.simulation
```
Comment (KmakD) [inline demo/simulation/compose.simulation-gpu.yaml:7]:
```suggestion
    image: husarion/panther:noetic-simulation
```

File: Dockerfile.simulation [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    mv panther_ros/panther_description/ src/panther_description && \
+    mv panther_ros/panther_gazebo/ src/panther_gazebo && \
```
Comment (KmakD) [inline Dockerfile.simulation:17]: This is ok, but it removes `panther_ros` from the path causing some of the demos not to work as they still contain volumes with it in. We can add that folder to path but I would personally change paths in volumes in demos.

**Review by KmakD — CHANGES_REQUESTED** [review]
Worth consideration adding `container_name` to all compose files.

**Review by KmakD — APPROVED** [review]

**Pull request merged**